tests/test_collection.py

```diff
@@ -18,21 +18,32 @@ def test_collection():
         "cif", "http://emmo.info/0.1/cif-ontology#"
     )
     triples = [
+        (STRUCTURE.name, DM.hasLabel, en("Strontium titanate")),
+        (STRUCTURE.masses, DM.hasUnit, Literal("u", datatype=XSD.string)),
         (STRUCTURE.symbols, MAP.mapsTo, EMMO.Symbol),
         (STRUCTURE.positions, MAP.mapsTo, EMMO.PositionVector),
         (STRUCTURE.cell, MAP.mapsTo, CIF.cell),
         (STRUCTURE.masses, MAP.mapsTo, EMMO.Mass),
     ]
 
     ts.add_triples(triples)
+
     assert set(ts.triples()) == set(triples)
 
     ts.remove(object=EMMO.Mass)
     assert set(ts.triples()) == set(triples[:-1])
 
-    # Test that we can initialise from an existing collection
-    coll = dlite.Collection()
-    for triple in triples:
-        coll.add_relation(*triple)
-    ts2 = Triplestore(backend="collection", collection=coll)
-    assert set(ts2.triples()) == set(triples)
+
+# For PR #160
```

Comment:
Because I somehow managed to mess up this PR with #160. The updates of `test_collection.py` should have been in #160. Updating the collection backend will be cleaned up in that PR. Sorry for the mess.

Reply:
```suggestion
# TODO: Fix handling of Literal in Collections (Issue #160, PR #165) and reactivate test.
```